server/entitymanager/entitymanager.go

```diff
@@ -1,39 +1,278 @@
-// Package entitymanager is an in-memory implementation of an entity manager that models an organization's inventory.
 package entitymanager
 
 import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/openconfig/bootz/proto/bootz"
+	epb "github.com/openconfig/bootz/server/entitymanager/proto/entity"
+	authz "github.com/openconfig/gnsi/authz"
+	certz "github.com/openconfig/gnsi/certz"
+	pathz "github.com/openconfig/gnsi/pathz"
+
+	log "github.com/golang/glog"
 	"github.com/openconfig/bootz/server/service"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/encoding/prototext"
 )
 
-// InMemoryEntityManager provides a simple in memory handler
-// for Entities.
-type InMemoryEntityManager struct{}
+type entityManager struct {
+	chassisConfigs []*epb.Chassis
```

Comment:
Perhaps could this be a map to make lookups easier? The key could be a struct like service.EntityLookup as it only contains manufacturer and serial.